README.md

```diff
@@ -1663,6 +1677,31 @@ elements:
           state: on
           state_not: off
       mediaLoaded: true
+    # Full form PTZ actions (only left button shown).
+  - type: custom:frigate-card-ptz
+    orientation: vertical
+    style:
+      transform: none
+      right: 20px
+      top: 180px
```

Comment:
```suggestion
    style:
      transform: none
      right: 5%
      top: 50%
```

Reply:
For some reason, 45% is what actually makes it properly centered vertically. Not sure whether this is a bug or not.